tasks/cops_documentation.rake

```diff
@@ -13,61 +13,8 @@ end
 desc 'Generate docs of all cops departments'
 task generate_cops_documentation: :yard_for_generate_documentation do
   RuboCop::Cop::Registry.with_temporary_global do
```

Comment:
I am not sure we need the `with_temporary_global` block – rubocop-factory_bot doesn’t have it.

Reply:
Indeed. I removed it.